Description: Extend list of the supported devices.
 commit:1b077906ea - Motorola Xoom (two devices)
 commit:509cd65c8f - Samsung YP-Q3
 .
 The patch for Nokia E7 comes from Launchpad bug#738185.
Origin: upstream, see above
Applied-Upstream: yes
---
 src/music-players.h |   11 +++++++++++
 1 file changed, 11 insertions(+)

--- libmtp.orig/src/music-players.h
+++ libmtp/src/music-players.h
@@ -217,6 +217,11 @@
   { "Samsung", 0x04e8, "YP-R0", 0x5125,
       DEVICE_FLAG_UNLOAD_DRIVER | DEVICE_FLAG_PLAYLIST_SPL_V1 |
       DEVICE_FLAG_UNIQUE_FILENAMES | DEVICE_FLAG_BROKEN_MTPGETOBJPROPLIST },
+  // From Manuel Carro
+  // Copied from Q2
+  { "Samsung", 0x04e8, "YP-Q3", 0x5130, DEVICE_FLAG_UNLOAD_DRIVER |
+      DEVICE_FLAG_OGG_IS_UNKNOWN | DEVICE_FLAG_BROKEN_MTPGETOBJPROPLIST |
+      DEVICE_FLAG_PLAYLIST_SPL_V1 },
   // From a rouge .INF file
   // this device ID seems to have been recycled for:
   // the Samsung SGH-A707 Cingular cellphone
@@ -764,6 +769,8 @@
   { "Nokia", 0x0421, "N8", 0x02fe, DEVICE_FLAG_NONE },
   // From: Lan Liu at Nokia <lan.liu@nokia.com>
   { "Nokia", 0x0421, "N8 (Ovi mode)", 0x0302, DEVICE_FLAG_NONE },
+  // From: Raul Metsma <raul@innovaatik.ee>
+  { "Nokia", 0x0421, "E7", 0x0335, DEVICE_FLAG_NONE },
   // https://sourceforge.net/tracker/index.php?func=detail&aid=2692473&group_id=8874&atid=358874
   { "Nokia", 0x0421, "5200", 0x0421, DEVICE_FLAG_NONE },
   // From: http://nds2.nokia.com/files/support/global/phones/software/Nokia_3250_WMP10_driver.inf
@@ -1155,6 +1162,10 @@
   // Reported by anonymous user
   { "Motorola", 0x22b8, "RAZR2 V8/U9/Z6", 0x6415,
       DEVICE_FLAG_BROKEN_SET_OBJECT_PROPLIST },
+  { "Motorola", 0x22b8, "Xoom (ID 1)", 0x70a8,
+      DEVICE_FLAG_BROKEN_MTPGETOBJPROPLIST },
+  { "Motorola", 0x22b8, "Xoom (ID 2)", 0x70a9,
+      DEVICE_FLAG_BROKEN_MTPGETOBJPROPLIST },
 
 
   /*
